python/hsfs/code.py

```diff
@@ -0,0 +1,75 @@
+#
+#   Copyright 2021 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import humps
+
+from hsfs import util
+
+
+class Code:
+    def __init__(
+        self,
+        commit_time,
+        application_id,
+        content=None,
+        feature_group_commit_id=None,
+        href=None,
+        expand=None,
+        items=None,
+        count=None,
+        type=None,
+    ):
+        self._commit_time = commit_time
+        self._application_id = application_id
+        self._feature_group_commit_id = feature_group_commit_id
+        self._content = "" if content is None else json.loads(content)
+
+    @classmethod
+    def from_response_json(cls, json_dict):
+        json_decamelized = humps.decamelize(json_dict)
+        # Currently getting multiple commits at the same time is not allowed
+        if json_decamelized["count"] == 0:
+            return None
+        elif len(json_decamelized["items"]) == 1:
+            return cls(**json_decamelized["items"][0])
```

Comment:
This method is not used as we don't support getting the code from the Hopsworks API, you can remove it.